P32-P33_P72-P73.s

```diff
@@ -42,20 +42,20 @@
 # MOD BY WHITE, P.	DATE 1 JUNE 67
```

Comment:
`WHITE, P.` should be `WHITE. P`
[Source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/0649.jpg)

Date section should be `1JUNE67`